README.md

```diff
@@ -1,89 +1,96 @@
 # Magento Developer Documentation
 
-Welcome! This site contains the latest Magento developer documentation for ongoing Magento 2 releases. For additional information, see our [Contribution Guide](https://github.com/magento/devdocs/blob/master/.github/CONTRIBUTING.md).
+Welcome! This repository contains the latest Magento developer documentation for ongoing Magento 2 releases. To learn more about contributing to Magento 2 code, see our [Contribution Guide](https://github.com/magento/devdocs/blob/master/.github/CONTRIBUTING.md).
 
-## Contributors
+## Contributing to the documentation
 
-Our goal is to provide the Magento community with comprehensive and quality technical documentation. We believe that to accomplish that goal we need experts from the community to share their knowledge with us and each other. We are thankful to all of our contributors for improving Magento documentation.
+Our goal is to provide the Magento community with comprehensive and quality technical documentation. We believe that to accomplish our goal we need experts from the community to share their knowledge with us and each other. We are thankful to all of our contributors for helping to improve the Magento documentation.
 
 ![DevDocs contributors](https://raw.githubusercontent.com/wiki/magento/magento2/images/dev_docs_contributors.png)
 
 ## Building this site
 
-The website uses [GitHub Pages](https://pages.github.com/) tools for building with the [Jekyll](https://jekyllrb.com/) static website generator on top of it.
-The toolkit requires a Ruby environment to be run.
+The website uses [GitHub Pages](https://pages.github.com/) tools for building with the [Jekyll](https://jekyllrb.com/) static website generator, which requires a Ruby environment.
 
-There are several options available depending on what type and level of virtualization you want to use:
+After verifying your [Ruby environment](#ruby-environment), continue with one of the following methods:
```

Comment:
After verifying your [Ruby environment](#ruby-environment), continue with one of the following build methods: